src/set_theory/game/pgame.lean

```diff
@@ -6,6 +6,7 @@ Authors: Reid Barton, Mario Carneiro, Isabel Longbottom, Scott Morrison
 import data.fin.basic
 import data.list.basic
 import logic.relation
+import logic.small
```

Comment:
Note that in the master you are based off, importing `logic.small` is quite expensive, as it imports all of the finiteness hierarchy. This is fixed in current master. But it is a good habit to check, e.g. using
```leanproject import-graph --to set_theory.game.pgame --reduce out.pdf```
that you are not doing too much damage to the import dependencies!

Once this is merged with current master, it should be okay, however; then `logic.small` becomes cheap again.